tests/unit/cli/test_cli.py

```diff
@@ -12,11 +12,44 @@
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
 
-from click.core import Group
+import click
+import pytest
+from click.testing import CliRunner
 
-from zenml.cli import cli
+from zenml.cli.cli import ZenMLCLI, cli
+from zenml.cli.formatter import ZenFormatter
+
+
+@pytest.fixture(scope="function")
+def runner(request):
+    return CliRunner()
 
 
 def test_cli_command_defines_a_cli_group() -> None:
     """Check that cli command defines a CLI group when invoked"""
-    assert isinstance(cli, Group)
+    assert isinstance(cli, ZenMLCLI)
+
+
+def test_cli(runner):
+    """Check that basic cli call works"""
+    result = runner.invoke(cli)
+    assert not result.exception
+    assert result.exit_code == 0
+
+
+def test_cli_help(runner):
+    """Check that help works"""
+    result = runner.invoke(cli, ["--help"])
+    assert not result.exception
+    assert "Available Commands By Group Category for Zenml:" in result.output
+    assert result.exit_code == 0
+
+
+def test_ZenMLCLI_formatter():
+    """
+    Test the ZenFormatter class.
+    """
+    zencli = ZenMLCLI()
+    context = click.Context(zencli)
+    formatter = ZenFormatter(context)
+    assert isinstance(formatter, ZenFormatter)
```

Comment:
What does this actually test? Possible could delete this?

Reply:
My idea behind this test is to verify that our new group object (CLI) has a ZenFormatter instance instead of the default HelpFormatter. but I think the test doesn't make sense.